content/21.nutraceuticals.md

```diff
@@ -289,7 +293,10 @@ These include polyphenols, lipid extracts, and tomato-based nutraceuticals among
 Melatonin is another supplement that has been identified as a potential antiviral agent against SARS-CoV-2 using computational methods [@doi:10.1038/s41421-020-0153-3], and it has also been highlighted as a potential therapeutic agent for COVID-19 due to its documented antioxidant, anti-apoptotic, immunomodulatory, and anti-inflammatory effects [@doi:10.3389/fnut.2020.583080; @doi:10.3390/antiox10010047; @doi:10.3390/nu13010047].
 Notably, melatonin, vitamin D and zinc were included in the treatment plan of the Former President of the United States upon hospitalization due to COVID-19 [@doi:10.1126/science.abf0974].
 These are just some of the many substances and supplements under investigation that currently have limited evidence to support their use or supplementation with the aim to prevent or treat COVID-19.
-While there is plenty speculation surrounding the use of supplements, a survey of 327,720 users of the COVID Symptom Study App found that the consumption of n-3 PUFA supplements, probiotics, multivitamins, and vitamin D was associated with a lower risk of SARS-CoV-2 infection in women but not men after adjusting for potential confounders [@doi:10.1101/2020.11.27.20239087].
+While there is plenty speculation surrounding the use of supplements by physicians and scientists, this has not stopped consumers from purchasing these products as one study in Poland has reported that searches for dietary supplements trended online at the start of the pandemic. 
+Additionally, supplement usage increased between the first and second wave of the pandemic . 
```

Comment:
I would recommend adding the citation in again!

Reply:
I think you put it in, do you want to add another reference to it? @rando2

Reply:
Oops! I didn't realize this was also the Polish study. In that case, I'll just suggest removing the final space!

```suggestion
Additionally, supplement usage increased between the first and second wave of the pandemic. 
```